includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -831,6 +831,8 @@ public function get_error_message( $error_code ) {
 				return __( 'Unable to receive access token because of an empty authorization code.', 'google-site-kit' );
 			case 'access_token_not_received':
 				return __( 'Unable to receive access token because of an unknown error.', 'google-site-kit' );
+			case 'access_denied':
+				return __( 'The Site Kit setup was interrupted because you did not grant the necessary permissions.', 'google-site-kit' );
```

Comment:
@felixarntz @aaemnnosttv we should avoid using "you" here, I would suggest to rephrase to 'The Site Kit setup was interrupted because the necessary permissions were not granted.'

Reply:
I agree. I had [suggested the same wording previously](https://github.com/google/site-kit-wp/issues/1192#issuecomment-603748461) so I assumed that this was intentional/preferred for some reason.

Reply:
```suggestion
				return __( 'The Site Kit setup was interrupted because the necessary permissions were not granted.', 'google-site-kit' );
```

Reply:
@ThierryA @aaemnnosttv I'm not sure about this one, why use passive here? If this happens, it is because _the user_ did not grant the necessary permissions. If we use passive, it sounds more like it could also be some application bug or "something went wrong", which it is not, and I think addressing the user clarifies that.

Reply:
After thinking more about this one, I see your point and I also miss read the original back and fourth comment on the issue. Generally using "you" is not great practice in error messages but here I think it is actually positive to let the user know that it is due to their action, meaning their are able to solve it too 😄 

Happy to change it back, my bad.